subaru_outback_2016_eyesight.dbc

```diff
@@ -36,17 +36,32 @@ BS_:
 BU_: XXX
 
 
+BO_ 884 DoorStatus: 8 XXX
+ SG_ DoorOpenFD : 24|1@0+ (1,0) [0|1] "Front Driver Door" XXX
+ SG_ DoorOpenFP : 25|1@0+ (1,0) [0|1] "Front Passenger Door" XXX
+ SG_ DoorOpenRP : 26|1@0+ (1,0) [0|1] "Rear Passenger Door" XXX
+ SG_ DoorOpenRD : 27|1@0+ (1,0) [0|1] "Rear Driver Door" XXX
+ SG_ DoorOpenHatch : 28|1@0+ (1,0) [0|1] "Rear Hatch" XXX
+
+BO_ 324 CruiseControl: 7 XXX
+ SG_ BrakeApplied : 8|1@0+ (1,0) [0|0] "" XXX
+ SG_ CruiseEnabled : 48|1@0+ (1,0) [0|0] "" XXX
+ SG_ BrakeStatus : 51|1@0+ (1,0) [0|0] "" XXX
+ SG_ CruiseButtons : 4|2@0+ (1,0) [0|3] "" XXX
```

Comment:
if you have confirmed that the cruise buttons are mutually exclusive, then you should add the list of values (VAL_) at the end of the dbc, like you did for brake applied etc...